UPGRADE-3.4.md

```diff
@@ -63,6 +63,12 @@ Debug
 
  * Support for stacked errors in the `ErrorHandler` is deprecated and will be removed in Symfony 4.0.
 
+EventDispatcher
+---------------
+
+ * Implementing `TraceableEventDispatcherInterface` without the method
+   `clearCalledListeners()` is deprecated and will be unsupported in 4.0.
```

Comment:
If we can remove this deprecation, that'd be better IMHO, we don't this method in the interface to method. The tag doesn't need it, and when we wire the service, we know their exact type, so their set of methods also, isn't it?
(same for the other interfaces).
*If* we need a contract for a "reset" method, I'd suggest adding a dedicate interface for it instead (there could be one per component when needed, didn't look precisely)

Reply:
The intention behind adding those methods to the interfaces was that each of them is designed to collect request-related data. There should not be a use-case of implementing them without the possiblity of resetting that data.

As it is implemented right now, the data collectors reset any helper classes they use for collecting their data. This way, you can easily reset the profiler, even if you're not using the `kernel.reset` feature.